ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-ref.yaml

```diff
@@ -17,5 +26,7 @@ ref:
   dependencies:
   - name: "release:latest"
     env: OPENSHIFT_INSTALL_RELEASE_IMAGE
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
```

Comment:
Is there a reason not to just reuse OPENSHIFT_INSTALL_RELEASE_IMAGE ?

Reply:
The build cluster requires cli and other functionalities specific to arch on which ci-operator runs on. Above configuration allows the cli to be used from x86 payload (release:latest) and only overrides the installation with arm payload.

Reply:
ahh, makes sense, thanks.